project/project.mjs

```diff
@@ -1,28 +1,54 @@
-/** Logic for the /project endpoint */
-import * as utils from '../utilities/shared.mjs'
-import * as fs from 'fs'
-
-export async function findTheProjectByID(id = null) {
```

Comment:
Why are you removing these?

Reply:
I have accommodated the old code and updated the code with my requirements from the issue